django/db/models/sql/compiler.py

```diff
@@ -407,6 +407,10 @@ def get_combinator_sql(self, combinator, all):
         parts = ()
         for compiler in compilers:
             try:
+                # Set the selects defined on the combined query by a call to
```

Comment:
Maybe you could try rephrasing this comment. I'm having trouble understanding it and suggesting an alternative.